DigitalLearningSolutions.Web/Views/Support/Faqs/Faqs.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.Support.Faqs
+@using Microsoft.Extensions.Configuration
+@model FaqsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/support/faqs.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "FAQs";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/Support/Shared/_SupportSideNavMenu.cshtml" model="@Model" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1 class="nhsuk-heading-xl">FAQs</h1>
+
+    <partial name="SearchablePage/_Search" model="Model" />
+    <input type="hidden" id="select-sort-by" value="@Model.SortBy" data-sort-by-multiple="true" />
+    <input type="hidden" id="select-sort-direction" value="@Model.SortDirection" />
+
+    @if (Model.NoDataFound) {
+      <p class="nhsuk-u-margin-top-4" role="alert">
+        <b>No FAQs found.</b>
+      </p>
+    } else {
+      <partial name="SearchablePage/_SearchResultsCount" model="Model" />
+
+      <div id="searchable-elements">
+        @foreach (var faq in Model.Faqs) {
+          <partial name="_SearchableFaqCard" model="faq" />
+        }
+      </div>
+    }
+    @if (Model.TotalPages > 1) {
+      <partial name="SearchablePage/_Pagination" model="Model" />
+    }
+
+    <nav class="side-nav-menu-bottom" aria-label="Bottom navigation bar">
+      <partial name="~/Views/Support/Shared/_SupportSideNavMenu.cshtml" model="@Model" />
+    </nav>
+  </div>
+</div>
+
+@section scripts {
+  <script src="@Url.Content("~/js/support/faqs.js")" asp-append-version="true"></script>
+}

```

Comment:
Might be nice to put this at the top since it's small, so it's obvious it's there.

Reply:
Done.